src/metrics/averageprecisionscore.jl

```diff
@@ -1,7 +1,7 @@
 import MLBase
 import StatsBase
 
-function _calculateaverageprecisionfromprecisionrecallcurve(
+function average_precision_score(
```

Comment:
maybe even shorter avg_precision(). score is kind of redundant

Reply:
Yep let's drop the score. I think I took the idea from scitkit-learn, who append "score" to the end of every score/metric function.